README.md

```diff
@@ -330,9 +328,55 @@ On top of that, we've added `gunicorn.conf.py` with some example settings.
 1. <https://devcenter.heroku.com/articles/python-gunicorn>
 1. <https://adamj.eu/tech/2019/09/19/working-around-memory-leaks-in-your-django-app/>
 1. <https://adamj.eu/tech/2021/12/29/set-up-a-gunicorn-configuration-file-and-test-it/>
-1. Worker settings - <https://docs.gunicorn.org/en/latest/settings.html#worker-processes>
+1. Worker settings - <https://docs.gunicorn.org/(en/latest/settings.html#worker-processes>
 1. A brief description of the architecture of Gunicorn - <https://docs.gunicorn.org/en/latest/design.html>
 
 ### AWS ECS
 
-*Coming soon*
+_Coming soon_
+
+## Linters and Code Formatters
+
+In all our Django projects we use:
+
+- [flake8](https://flake8.pycqa.org/en/latest/) - a linter that ensures we follow the PEP8 conventions
+- [black](https://github.com/psf/black) - a code formatter that ensures we have the same code style throughout the whole project
+- [isort](https://github.com/PyCQA/isort) - a code formatter that ensures we have the same imports style throughout the whole project
+- [pre-commit](https://pre-commit.com/) - a tool that triggers the linters before each commit
+
+To make sure all of the above tools work in symbiosis, you'd need to add some configuration:
+
+1. Add `.pre-commit-config.yaml` file to the root of your project. Here you can add the instructions for `pre-commit`
+2. Add `pyproject.toml` file to the root of your project. Here you can add the `black` config. **NOTE:** `black` does not respect any other config files.
+3. Add the following to `setup.cfg` for the `isort` config:
+
+```
+[isort]
+profile = black
+```
+
+This will tell `isort` to follow the `black` guidelines.
+
+```
+[isort]
+filter_files = true
+skip_glob = */migrations/*
+```
+
+This will tell `pre-commit` to respect the `isort` config.
```

Comment:
Maybe we should add `flake8` here too, since it is also configured in `setup.cfg` 🤔